src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +18,6 @@
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
Make sure to handle any potential exceptions that may occur during the execution of the CompletableFuture tasks. This could be done by adding exceptionally() or handle() to the CompletableFuture chain.

Reply:
Sure, this is the task requirement too.